cache/clock_cache.h

```diff
@@ -59,100 +61,136 @@ struct ClockHandle {
   Cache::DeleterFn deleter;
   uint32_t hash;
   size_t total_charge;  // TODO(opt): Only allow uint32_t?
-  // The number of external refs to this entry.
-  uint32_t refs;
+  std::array<char, kCacheKeySize> key_data;
+
+  static constexpr int kExternalRefsOffset = 0;
```

Comment:
nit: int --> uint32_t ?

Reply:
I'd actually also use a smaller int type. @pdillinger I remember you made a comment (which I can't find) in a previous PR about int vs. smaller integral types. Would you be so kind to explain the rationale? Is it to avoid accidental overflows? I can't find an answer in Google's C++ style guide.

Reply:
Small types like uint8_t or uint16_t can make sense for small constants because they are implicitly convertible to int and to larger unsigned types without compiler warnings.

Reply:
Thank you.